src/commands/build.ts

```diff
@@ -32,6 +32,21 @@ export class BuildCommand implements Command {
 
   args = [
     {
+      name: 'js-minify',
```

Comment:
We can, but the pattern had been set up with that goal in mind. If we defer the discussion can we stick with the current pattern?